tests/zfs-tests/tests/functional/pool_checkpoint/pool_checkpoint.kshlib

```diff
@@ -154,13 +154,18 @@ function setup_nested_pools
 
 function cleanup_nested_pool
 {
-	log_must zpool destroy $NESTEDPOOL
+	if poolexists $NESTEDPOOL; then
+		log_must zpool destroy $NESTEDPOOL
```

Comment:
Optionally `destroy_pool` instead here and below.

Reply:
That would work, though if you don't mind I think we have just leave it as is.